src/components/Sidenav/StudioSidenav/StudioSidenav.tsx

```diff
@@ -60,12 +59,13 @@ export const StudioSidenav: React.FC = () => {
       openWarningDialog({
```

Comment:
Do we still need that warning though? You don't lose anything by opening a new tab

Reply:
right, I guess we don't need this anymore.

Reply:
removed